data/presets/traffic_sign/hazard.json

```diff
@@ -0,0 +1,16 @@
+{
+    "fields": [
+        "hazard"
+    ],
+    "geometry": [
+        "point",
+        "vertex"
+    ],
+    "tags": {
+        "traffic_sign": "hazard"
```

Comment:
Will these presets automatically inherit fields from the existing Traffic Sign preset? The Direction field would be pretty useful.